app/code/Magento/Backend/view/adminhtml/web/js/store-switcher.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'jquery',
+    'mage/translate',
+    'Magento_Ui/js/modal/confirm'
+], function(jQuery, $t, confirm) {
+    'use strict';
+
+    (function($) {
```

Comment:
I have feeling that it should be inside the `return function(config) { ...`. What do you thing?